demo-app/src/runtime.rs

```diff
@@ -38,6 +38,7 @@ use sov_modules_macros::{DispatchCall, DispatchQuery, Genesis, MessageCodec};
 /// Similar mechanism works for queries with the difference that queries are submitted by users directly to the rollup node
 /// instead of going through the DA layer.
 #[derive(Genesis, DispatchCall, DispatchQuery, MessageCodec)]
+#[serialization(borsh::BorshDeserialize, borsh::BorshSerialize)]
```

Comment:
Problem: with the current setup borsh is sill required by other components

Reply:
Yes, that's fine. At some point, we want to use our own traits for `serialization/deserialization` everywhere and we will implement the traits by default with borsh (but we will be able to switch to another format if we want to.) This PR is an important step on the way.

Reply:
Are you sure pipelines work correctly as this PR is green, while this test should fail?